.travis.yml

```diff
@@ -11,7 +11,7 @@ _install: &_install
   - pip install -r requirements.txt codecov
   - pip install -e .
 _coverage: &_coverage
-  - SCRIPT="coverage run -m unittest discover"
+  - SCRIPT="python3 -m unittest discover shcomplete/"
```

Comment:
Have you tried `coverage run -m unittest discover shcomplete` ? You do not collect any coverage information in your current proposal.